### PR TITLE
[API-13803]testing getting tag from git

### PR DESCRIPTION
**Pull request by mhyder1**
# Proposed Changes

Describe the big picture of your changes here.

## Checklist

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

- [ ] My changes generate no new warnings
- [ ] Logging has been added where applicable (Error handling, user-driven events, API requests/responses)

- [ ] I have added tests that prove my fix is effective or that my feature works (if applicable)
- [ ] Tests meet the test coverage threshold

## Reviewer Checklist

- [ ] Is the code easy to understand?
- [ ] Are the changes well documented?

- [ ] Can you think of any use case in which the code does not behave as intended?
- [ ] Can you think of any inputs or external events that could break the code?

- [ ] Should any logging or debugging information be added or removed?

## Further Comments

If this is a relatively large or complex change, kick off the discussion by explaining how you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: .github/workflows/cicd.yml [hunk 1]
```diff
@@ -84,5 +85,6 @@ jobs:
         uses: ./.github/actions/send-deployment-webhook
         with:
           ENV: "dev"
+          VERSION: '1.2.3.4'
```
Comment (keyluck) [inline .github/workflows/cicd.yml:88]: Does this input need to be added to the composite action?

File: .github/actions/deploy-backstage-app/action.yml [hunk 2]
```diff
@@ -84,6 +87,10 @@ runs:
       fi
       echo "::set-output name=sha::$prefix"
     shell: bash
+  - name: Set version number
+    run: |
+      echo "::set-output name=VERSION::1.2.3.4"
```
Comment (keyluck) [inline .github/actions/deploy-backstage-app/action.yml:92]: Is this output being used at all?

File: packages/app/src/components/versionAndEnv/VersionAndEnv.tsx [hunk 3]
```diff
@@ -1,26 +1,28 @@
...
   data?: AppData;
 }
 
 export const VersionAndEnv: FC = (): any => {
-  const { version } = versionNumber;
   const apiConfig: ConfigVersion = useApi(configApiRef);
-
   const environment = apiConfig?.data?.auth?.environment;
-
+  const version = apiConfig?.data?.app?.version;
```
Comment (keyluck) [inline packages/app/src/components/versionAndEnv/VersionAndEnv.tsx:25]: What happens if no tag exists? Is this nil or '0.0.0'? Is it supposed to be a truncated commit sha?

File: .github/workflows/version-delete.yaml [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+  workflow_dispatch:
+jobs:
+  get-tag:
+    runs-on: ubuntu-latest
+    # env:
+    #   WORKFLOW_TOKEN: ${{secrets.OCTOKIT_GITHUB_WORKFLOW_TOKEN}}
```
Comment (KaemonIsland) [inline .github/workflows/version-delete.yaml:12]: Should these be removed?

File: .github/workflows/version-delete.yaml [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 14
+      # - name: Analyze Tags
+      #   uses: Dotunj/github-action-analyze-tags@v0.2.0
```
Comment (KaemonIsland) [inline .github/workflows/version-delete.yaml:21]: Same here

File: .github/workflows/version-delete.yaml (hunk 4 above)
Comment (mhyder1) [reply .github/workflows/version-delete.yaml:12]: yes. I was just using it for testing. will delete before merging

File: .talismanrc [hunk 6]
```diff
@@ -38,4 +38,9 @@ fileignoreconfig:
   checksum: c9203a04d4c35b249d5bc85a77d6c09e1511d98abed33f9cbf867031a6af5365
 - filename: docs/dev-deploy.md
   checksum: 70b93fd00c62efd78dbcc1ca3a0b1c7327a0ad330a1a70778e80be756061383f
+version: ""
```
Comment (keyluck) [inline .talismanrc:41]: By the way, when you update the `.talismanrc` you don't have to keep adding sections you can just overwrite the existing 'fileignoreconfig' section with the output in your terminal.

Comment (mhyder1) [reply .talismanrc:41]: I was wondering about that. Thanks

File: packages/app/src/components/versionAndEnv/VersionAndEnv.tsx (hunk 3 above)
Comment (mhyder1) [reply packages/app/src/components/versionAndEnv/VersionAndEnv.tsx:25]: you're right. fixing it.

**Pull request merged**